.github/workflows/version-bump.yml

```
@@ -0,0 +1,67 @@
+---
+name: Version Bump
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_number:
+        description: "New Version"
+        required: true
+
+jobs:
+  bump_props_version:
+    name: "Create version_bump_${{ github.event.inputs.version_number }} branch"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Branch
+        uses: peterjgrainger/action-create-branch@v2.0.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          branch: "version_bump_${{ github.event.inputs.version_number }}"
+
+      - name: Checkout Branch
+        uses: actions/checkout@v2
+        with:
+          ref: version_bump_${{ github.event.inputs.version_number }}
+
+      - name: Bump Version - Props
+        uses: bitwarden/gh-actions/version-bump@0c263b3963211ccaf5804313c3b3a0bcc52d4b19
+        with:
+          version: ${{ github.event.inputs.version_number }}
+          file_path: "./Directory.Build.props"
+
+      - name: Commit files
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git commit -m "Bumped version ${{ github.event.inputs.version_number }}" -a
+
+      - name: Push changes
+        uses: ad-m/github-push-action@master
```

I think a simple run action might be better suited here instead of using a third party action. I don't believe the token is needed. See the [web release](https://github.com/bitwarden/web/blob/221931ecaa764b91929eef9ab8a9ab703b5ce274/.github/workflows/release.yml#L97) for reference: 
```suggestion
        run: git push -u origin version_bump_${{ github.event.inputs.version_number }}
```